packages/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -40,6 +40,22 @@ const dropdownStyles: ComponentSlotStylesInput<DropdownPropsAndState, DropdownVa
     }),
   }),
 
+  clearIndicator: ({ variables: v }): ICSSInJSStyle => ({
+    alignItems: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+
+    backgroundColor: 'transparent',
+    cursor: 'pointer',
+    userSelect: 'none',
+
+    margin: 0,
+    position: 'absolute',
+    right: pxToRem(5),
```

Comment:
can we extract these in common styles for `clearIndicator` and `toggleIndicator`?

Reply:
ah...just saw your comment https://github.com/stardust-ui/react/pull/885#discussion_r255912391

I'd say to extract in one function

Reply:
Done 👍